understanding/20/timing-adjustable.html

```diff
@@ -40,6 +40,8 @@ <h2>Intent of Timing Adjustable</h2>
 
       <p>In some cases, however, it is not possible to change the time limit (for example, for an auction or other real-time event) and exceptions are therefore provided for those cases.</p>
       
+      <p>Not all content that operates on a timer requires that the timing is adjustable. For example, a web application such as an email client provides notification of new email arriving in toast messages in the lower right-hand side of the interface, and the toast messages disappear after 5 seconds. The users are able to verify the arrival of email through other means, such as viewing the Inbox, so the disappearance of the toast message does not set a time limit on the end user's ability to verify if new mail has arrived. If the messages conveyed via toast could not be verified separately, then the message would need to meet this Success Criterion in order to provide the user with sufficient time to access the information.</p>
```

Comment:
following on from Patrick's suggestion:
The following changes 
- the first instance of "The users" to just "Users"
- the second instance "the end user's" I change to "their" instead of Patrick's suggestion, since the sentence starting talking about users plural, and then switched to a single user.  
- the third instance of "the user" was changed to "users" to maintain parity with the previous plural mention of them.

Additionally changed "If the messages conveyed via toast..." to "If the toast messages..." to be consistent with the first mention of "toast messages", and because "conveyed via toast" is making it seem like "toast" is more than just a design/dev jargon term for a JS powered UI pattern, and an actual mechanism for message delivery.

Finally, changed "...then the message would need...." to "then each message would need..." because the sentences was starting to talk about messages plural and then changed to the singular.

```suggestion
      <p>Not all content that operates on a timer requires that the timing is adjustable. For example, a web application such as an email client provides notification of new email arriving in toast messages in the lower right-hand side of the interface, and the toast messages disappear after 5 seconds. Users are able to verify the arrival of email through other means, such as viewing the Inbox, so the disappearance of the toast message does not set a time limit on the their ability to verify if new mail has arrived. If the toast messages could not be verified separately, then each message would need to meet this Success Criterion in order to provide users with sufficient time to access the information.</p>
```

Reply:
agree with making it sound clearer/simpler/more direct